src/common/containers/landing.js

```diff
@@ -60,40 +59,51 @@ class Landing extends Component {
           </div>
         </div>
 
-        <section className={ `${styles.section} ${styles.sectionNoBorder} ${containerStyles.lightStrip}` }>
-          <div className={ `${styles.row} ${containerStyles.wrapper}` }>
-            <ListDividedState className={ styles.rowItemGrow }>
-              <li>Scale to millions of installs</li>
-              <li>Available on all clouds and Linux OSes</li>
-              <li>No need for build infrastructure</li>
-            </ListDividedState>
-
-            <ListDividedState className={ styles.rowItemGrow }>
-              <li>Automatic updates for everyone</li>
-              <li>Roll back versions effortlessly</li>
-              <li>FREE for open source projects</li>
-            </ListDividedState>
+        <section className={ `${styles.section} ${styles.sectionNoBorder}` }>
+          <div className={ `${containerStyles.wrapper}` }>
+            <HeadingTwo className={ styles.heading}>
```

Comment:
I'm not a big fan of passing arbitrary styles to Vanilla components (because it means that we have some exception from the regular styles). But it seems that landing page headings have more white space.

In such case I would prefer some more descriptive class name, to be clear that it's some specific version of the heading. So maybe `.landingHeading`?

And BTW `{ styles.heading}` could use a trailing space.